Source/Core/OrientedBoundingBox.js

```diff
@@ -0,0 +1,302 @@
+/*global define*/
+define([
+        './Cartesian2',
+        './Cartesian3',
+        './Cartographic',
+        './defaultValue',
+        './defined',
+        './DeveloperError',
+        './Ellipsoid',
+        './Intersect',
+        './Plane',
+        './Rectangle',
+        './Math',
+        './Matrix3'
+    ], function(
+        Cartesian2,
+        Cartesian3,
+        Cartographic,
+        defaultValue,
+        defined,
+        DeveloperError,
+        Ellipsoid,
+        Intersect,
+        Plane,
+        Rectangle,
+        CesiumMath,
+        Matrix3) {
+    "use strict";
+
+    /**
+     * Creates an instance of an OrientedBoundingBox.
+     * An OrientedBoundingBox model of an object or set of objects, is a closed volume (a cuboid), which completely contains the object or the set of objects.
```

Comment:
Do you want to rewrite this old doc?  It is not useful for our users.  Just say it is convex and can provide a tighter fix than spheres and AABBs.